test/cmd/apply.sh

```diff
@@ -477,7 +477,7 @@ __EOF__
   local tries=5
   for i in $(seq 1 $tries); do
       local output
-      output=$(kubectl "${kube_flags[@]:?}" api-resources --api-group mygroup.example.com -oname)
+      output=$(kubectl "${kube_flags[@]:?}" api-resources --api-group mygroup.example.com -oname || true)
```

Comment:
this loops never fails now if the api group doesn't exist, I think you should add a check with a bool or the number of tries

Reply:
isn't the purpose not failing?, because this line already checks the string https://github.com/kubernetes/kubernetes/pull/113708/files#diff-46e80bcde84ba84b7739a9124ac5ff40986c2d4ee7122bd5e43bb69dd235795eR481

Reply:
you are right, there should be a verification if tries=5

Reply:
👍

Reply:
This is just waiting for the resource to be available, it will fail as soon as it leaves the loop with a rightful error as we're trying to apply a CR of that type?

Reply:
oh, I see , thanks for the clarification and sorry for the noise